build.gradle

```diff
@@ -90,44 +90,103 @@ subprojects {
 
         mavenCentral()
         if (!project.block_external_repositories) {
-            maven {
-                url = "https://maven.realrobotix.me/master/"
-            }
             mavenLocal()
             maven {
                 name = "ParchmentMC"
                 url = "https://maven.parchmentmc.org"
             }
-            maven { url = "https://cursemaven.com" }
-            maven { url = "https://maven.terraformersmc.com/releases/" } // Mod Menu
+            maven {
+                url = "https://cursemaven.com"
+                content { includeGroup("curse.maven") }
```

Comment:
I do not prefer compressed style here, since it make desync `content` block style.